pep-0001.txt

```diff
@@ -455,15 +460,26 @@ Each PEP should have the following parts:
    The final implementation must include test code and documentation
    appropriate for either the Python language reference or the
    standard library reference.
-
-9. How to Teach This -- For a PEP that adds new functionality or changes
-   language behavior, it is helpful to include a section on how to
-   teach users, new and experienced, how to apply the PEP to their
-   work.
-
-   This section may include key points and recommended documentation
-   changes that would help users adopt a new feature or migrate their
-   code to use a language change.
+   
+9. Rejected Ideas -- Throughout the discussion of a PEP, various ideas
+   will be proposed which are not accepted. Those rejected ideas should
+   be recorded along with the reasoning as to why they were rejected.
+   This boths helps record the thought process behind the final version
+   of the PEP as well as preventing people from bringing up the same
+   rejected idea again in subsequent discussions.
+
+10. Open Issues -- While a PEP is in draft, ideas can come up with warrant
+    further discussion. Those to-be-discussed ideas should be recorded
+    so people know that they are being thought about but do not have a
+    concrete resolution. This helps make sure all issues required for the
+    PEP to be ready for consideration are complete as well as prevent people
+    from bringing them up independently.
+    
+11. References -- A collection of URLs used as references through the PEP. 
+   
+12. Copyright/public domain -- Each PEP must either be explicitly
+    labeled as placed in the public domain (see this PEP as an
+    example) or licensed under the `Open Publication License`_.
 
```

Comment:
Should we perhaps make a security section mandatory too? Or at least mention that if there *are* any security concerns they should be collected in a separate section? (Because if most PEPs don't have security issues, requiring everyone to have a section "Security concerns" that says "There are none" is a bit silly.)

Reply:
Similar to Section 7: How to Teach, there could be a `Section x: Deployment, Configuration, and Security` that highlights any specific deployment, configuration, or security processes or best practices when used.